README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
As we use utilities (that aren't libraries tracked by Python or Node), can you keep track of them in the readme? Eg, the Mapbox CLI.